src/spec.ts

```diff
@@ -336,6 +361,9 @@ function fieldDefIndex(spec: GenericSpec<GenericUnitSpec<any, any>>, dict: any =
   } else if (isFacetSpec(spec)) {
     accumulate(dict, vlEncoding.fieldDefs(spec.facet));
     fieldDefIndex(spec.spec, dict);
+  } else if (isRepeatSpec(spec)) {
+    accumulate(dict, vlEncoding.fieldDefs(spec.spec));
+    fieldDefIndex(spec.spec, dict);
```

Comment:
We actually use this in tooltip.  Maybe we don't really instantiate the model to make this method correct.  Maybe file an issue to fix this later?